px-map-behavior-popup.es6.js

```diff
@@ -247,8 +247,9 @@
 
       const maxWidth = 400;
       const minWidth = 300;
+      const maxHeight = 300;
```

Comment:
Not sure about making this a static value by default. What is the map is large and the developer wants to allow popups to be bigger than 300px tall? Perhaps this should be a new, opt-in property passed the developer when they use the popup components? Let me know your thoughts.